src/engine/server/server.cpp

```diff
@@ -1915,5 +1902,5 @@ void CServer::SendInitialData(int ClientID)
 
 bool CServer::GetClientProtocolCompatibility(int ClientID, int Protocol) const
 {
-	return (m_aClients[ClientID].m_Protocol == MODAPI_CLIENTPROTOCOL_TW07MODAPI);
+	return (m_aClients[ClientID].m_Protocol == Protocol);
```

Comment:
Remove this, the old version is correct. This function check if the protocol is compatible with GUI, Sprites, ...